.github/workflows/check-links.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     - uses: actions/checkout@v1
     - name: Link Checker
       id: lc
-      uses: peter-evans/link-checker@v1
+      uses: peter-evans/link-checker@v1.2.2
```

Comment:
This shouldn't be necessary if I understand https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses correctly:

> Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.

Might be worth switching to major versions everywhere?

@ckotzbauer WDYT?

Reply:
I would also prefer to only specify the major version. Only if it's really necessary to do otherwise.